src/core/print.ml

```diff
@@ -231,7 +220,7 @@ and pp_term : term pp = fun ppf t ->
     | Type        -> out ppf "TYPE"
     | Kind        -> out ppf "KIND"
     | Symb(s)     -> pp_sym ppf s
-    | Meta(m,e)   -> out ppf "%a%a" pp_meta m pp_env e
+    | Meta(m,e)   -> out ppf "%a.%a" pp_meta m pp_env e
```

Comment:
The dot should be included in pp_env since it is also needed for Patt. Then, dtrees.expected should be updated accordingly.

Reply:
Fixed in https://github.com/Deducteam/lambdapi/pull/812/commits/fecec3b56ce9160ac12f008e234cc6a69a4a1bb1.